REQUIRE

```diff
@@ -3,3 +3,8 @@ julia 0.6
 JuMP 0.17 0.19-
 PowerModels 0.5 0.6-
 Compat 0.17
+
+DataFrames 0.11
+Memento 0.1.1
+Unitful 0.7.0
+Missings 0.2.6
```

Comment:
I try to avoid package dependencies, unless the added value is notable and the packages are stable and well maintained.  Can you comment on need/stability of these new packages?

For example, is there a compelling reason to use Missing instead of Nullable?

Reply:
`Missing` is supposed to be the new Julia standard for missing values. The new `DataFrames` does not use `NA` anymore. IIRC they dropped nullables as well.

Memento is just a way of suppressing the warnings we get when loading Matpower cases. Whenever the case is reasonably large, the amount of warnings can cover and hide errors/stacktraces. 

DataFrames is used as an easier to read/manipulate version of the network (that avoids the need to loop over keys when you want all of the load values, for instance).

Unitful is necessary in order to add units to the quantities.

Reply:
K, thanks for the explanation.